styles.css

```diff
@@ -0,0 +1,113 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: montserrat;
+    color: #fff;
+}
+
+/* **Header** */
+header {
+    background-color: #46424D;
+    height: 60px;
+}
+h1 {
+    padding: 1% 0;
+    color: #fff;
+    font-size: 1.5em;
+    font-weight: 300;
+    text-align: center;
+}
+main {
+    background-color: #2b2733;
+    height: 95vh;
```

Comment:
The page looks a little funky on my laptop. I think we can change some formatting in the css to fix it. One of them is using % instead of vh.

Reply:
Agreed.